README.md

```diff
@@ -51,7 +51,21 @@ export VAULT_AUTO_UNSEAL=true
 export DPS_VERSION=2.0.0.0-142
 ```
 
-In order to run Cloudbreak, Periscope, Datalake, FreeIPA, Redbeams, Environment, Auth Mock, IDBroker Mapping Management, and Environments2 API services locally (from IDEA or the command line), put this into your `Profile`:
+If you are using AWS, then also add the following lines, substituting your workload account id, and the AWS credentials that you have created for the CB role.
+
+```
+export CB_AWS_ACCOUNT_ID="THE_WORKLOAD_ACCOUNT_ID_PROBABLY_069336058373"
```

Comment:
?

Reply:
Thanks for catching this.

I have also added an unrelated fix for the cloudbreak CLI command.